_episodes/03-matplotlib.md

```diff
@@ -228,7 +228,7 @@ formats, including SVG, PDF, and JPEG.
 > > matplotlib.pyplot.show()
 > > ~~~
 > > {: .language-python}
-> ![Plot with step lines](../fig/inflammation-01-line-styles.svg)
+> ![Three line graphs, with step lines connecting the points, showing the daily average, maximum and minimum inflammation over a 40-day period.](../fig/inflammation-01-line-styles.svg)
```

Comment:
To keep this line under 100 characters, would you mind splitting the text on to a second line?

Reply:
Thanks for pointing that out. Fixed by cf41831.